app/domain/export/pdf/messages/letter_with_invoice.rb

```diff
@@ -37,5 +38,14 @@ def render_payment_slip(pdf, recipient)
     def cursors
       @cursors ||= {}
     end
+
+    private
+
+    def render_donation_confirmation(pdf, recipient)
+      # rubocop:disable LineLength
+      Export::Pdf::Messages::LetterWithInvoice::DonationConfirmation.new(pdf, @letter, recipient, {}).render
+      # rubocop:enable LineLength
```

Comment:
Da sich die Klasse im gleichen Modul-Namespace befindet, kannst du das auch einfach kürzer schreiben:
```suggestion
      LetterWithInvoice::DonationConfirmation.new(pdf, @letter, recipient, {}).render
```
Oder falls das nicht funktioniert würde ich lieber die Argumente auf eine neue Zeile umbrechen statt Rubocop zu disablen.